lgc/include/lgc/state/AbiMetadata.h

```diff
@@ -196,4 +196,38 @@ constexpr unsigned int mmSPI_SHADER_USER_DATA_HS_0 = 0x2D0C; // For GFX9+, Used
 constexpr unsigned int mmSPI_SHADER_USER_DATA_PS_0 = 0x2C0C;
 constexpr unsigned int mmSPI_SHADER_USER_DATA_VS_0 = 0x2C4C;
 
+// Other SPI register numbers in PAL metadata
+constexpr unsigned int mmPA_CL_CLIP_CNTL = 0xA204;
+
+// Register bitfield layout.
+
+union PA_CL_CLIP_CNTL {
```

Comment:
Maybe we have more register definitions moved to here. So could add a comment to highlight (possible add more comments if there are special notes).

```
// Register bitfield layout.

// PA_CL_CLIP_CNTL
union PA_CL_CLIP_CNTL {
}

// ...
union Xxx {
}
```